doc/man3/CRYPTO_THREAD_run_once.pod

```diff
@@ -25,6 +32,28 @@ CRYPTO_atomic_add, CRYPTO_atomic_or, CRYPTO_atomic_load - OpenSSL thread support
                       CRYPTO_RWLOCK *lock);
  int CRYPTO_atomic_load(uint64_t *val, uint64_t *ret, CRYPTO_RWLOCK *lock);
 
+ void CRYPTO_mem_barrier(void);
+
+ int CRYPTO_THREAD_cap(OSSL_LIB_CTX *ctx, int max_threads);
+ int CRYPTO_THREAD_disable(OSSL_LIB_CTX *ctx);
+ int CRYPTO_THREAD_enable(OSSL_LIB_CTX *ctx, int max_threads);
+ int CRYPTO_THREAD_enabled(OSSL_LIB_CTX *ctx);
+ int CRYPTO_THREAD_spawn_worker(OSSL_LIB_CTX *ctx, CRYPTO_WORKER_CALLBACK cb);
```

Comment:
The definition of `CRYPTO_WORKER_CALLBACK` should be included int the documentation

Reply:
Would mentioning it in `DESCRIPTION` be acceptable? Cannot add it into `SYNOPSIS` without adding it into `NAME`, which causes `doc-nits` to fail.